Files/Helpers/FileListCache.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Files.Filesystem;
+using Microsoft.Extensions.Caching.Memory;
```

Comment:
Are .NET Platform extensions allow in UWP apps?

Reply:
As i wrote earlier - i have no prior experience with UWP so i'm not completely sure, but i don't see any reason why shouldn't it be allowed. It's targeting .NET Standard 2.0, so i believe it should be working practically everywhere. And for sure it's working on my machine :)

Reply:
@duke7553 They are allowed